server/forms/sentence-plan/v1.0/effects/loadOrCreatePlanByOasys.ts

```diff
@@ -13,6 +13,10 @@ export const loadOrCreatePlanByOasys = (deps: SentencePlanEffectsDeps) => async
   const user = context.getState('user')
   const session = context.getSession()
 
+  if (user) {
+    context.setData('user', user)
+  }
```

Comment:
Good workaround for the moment. I do actually have a PR to raise eventually where I've added a new `Request.Xyz()` that you can use in form configs etc to get stuff like state/session directly. Would be like `Request.State('user')` as the replacement for this!